apps/supernova/src/lib/createFiltersSlice.jsx

```diff
@@ -28,11 +28,21 @@ const parsePredefinedFilters = (predefinedFilters) => {
 const parseInitialFilters = (initialFilters, filterLabels) => {
   if (!initialFilters) return initialFiltersState.initialFilters
 
-  if (typeof initialFilters !== "object") {
+  if (typeof initialFilters !== "object" || initialFilters === null) {
```

Comment:
**(Non-blocking suggestion)**

Safer coverage?

```suggestion
  if (!initialFilters || typeof initialFilters !== "object")
```

Reply:
its not safer because undefined is not an object.

Reply:
cool @guoda-puidokaite 

!initialFilters checks if initialFilters is falsy (i.e., null, undefined, false, 0, NaN, or an empty string).

Reply:
> its not safer because undefined is not an object.

Not sure what you mean by this? I just think additionally to checking for `null`, we could cover all `falsy` values. But maybe not necessary here. You know code more @TilmanHaupt @ArtieReus

Reply:
All other values are not objects.

Reply:
Ahhh, duh. Nice.